src/common/codeUtil.ts

```diff
@@ -48,4 +51,85 @@ export class CodeUtil {
 
         return regs;
     }
+
+    /** Align variable assignment by = operator in selected text
+     * @param selection Text selection in editor
+     */
+    public static alignTextAssignOperator(selection: Selection): string {
+        const document = vscode.window.activeTextEditor.document;
+        let maxPosition = 0; // Right-most = operator position in line from all assignments
```

Comment:
`/**` comments show up when you hover over the variable :)

```suggestion
        const document = vscode.window.activeTextEditor.document;
        /** Right-most `=` operator position in line from all assignments */
        let maxPosition = 0;
```